apps/teams-test-app/src/components/VisualMediaAPIs.tsx

```diff
@@ -0,0 +1,104 @@
+import { visualMedia } from '@microsoft/teams-js';
+import React, { ReactElement } from 'react';
+
+import { ApiWithoutInput, ApiWithTextInput } from './utils';
+import { ModuleWrapper } from './utils/ModuleWrapper';
+
+const imagesHelper = (medias: visualMedia.VisualMediaFile[]): string => {
+  let message = '';
+  for (let i = 0; i < medias.length; i++) {
+    const media: visualMedia.VisualMediaFile = medias[i];
+    message +=
+      '[content: ' +
+      JSON.stringify(media.content) +
```

Comment:
Json.stringify could handle json structure to convert it to a string. Maybe you could leverage on it without using for loop.

Reply:
Tried directly using Json.stringify, and it turned out [Object] as result.